mock_connection_id_generator_test.go

```diff
@@ -0,0 +1,15 @@
+package quic
```

Comment:
By convention, the `mock_xxx_test.go` files are all auto-generated and use gomock. Can you move this to the test files please?

Reply:
yes. missed that. moved.